pkg/client/didexchange/client.go

```diff
@@ -78,3 +78,18 @@ func (c *Client) QueryConnectionByID(id string) (*QueryConnectionResult, error)
 		ConnectionID: uuid.New().String(), CreateTime: time.Now(),
 	}, nil
 }
+
+// AcceptExchangeRequest accept a stored exchange request
+func (c *Client) AcceptExchangeRequest(id string) (*ExchangeResponse, error) {
+
+	// TODO use c.didexchangeSvc.Handle to be called once did exchange service is
+	//  ready to handle accept exchange request (#238)
+
+	// TODO returning sample response since
+	//  event listening/handling with DID exchange service needs to be implemented (#198)
```

Comment:
what's the message type(event) expected here ?

Reply:
Message Type still need to be added for this, more details in  #238